macaroon_service_test.go

```diff
@@ -24,14 +25,21 @@ func TestMacaroonServiceMigration(t *testing.T) {
 	require.NoError(t, err)
 	defer os.RemoveAll(tempDirPath)
 
+	db, err := kvdb.GetBoltBackend(&kvdb.BoltBackendConfig{
+		DBPath:     tempDirPath,
+		DBFileName: "macaroons.db",
+		DBTimeout:  defaultDBTimeout,
+	})
+	require.NoError(t, err)
+	rks, err := macaroons.NewRootKeyStorage(db)
```

Comment:
The convenience function mentioned above would be handy here.

Reply:
Added!